Formula/tracker.rb

```diff
@@ -39,7 +39,8 @@ class Tracker < Formula
   uses_from_macos "libxml2"
 
   def install
-    args = std_meson_args + %w[
+    args = %w[
+      --force-fallback-for=gvdb
```

Comment:
I think we should declare this as a `resource`, instead of letting `meson` download it on its own, so we can cache it properly.

See `resource("rs")` in my GStreamer PR for an example.

Reply:
It's actually not downloaded by meson, it's a git submodule so it is cached I believe since we check submodules out when the software spec is a git one.

Reply:
Yea, it would be downloaded by Meson if we used the tarball. We should switch back to the tarball (better for caching) and add the resource.